src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -312,9 +326,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
             else if (!_settings.BackgroundImage().empty())

```

Comment:
I'm not sure this will update the background image+acrylic scenario properly - looks like it'll only update the acrylic in that scenario

Reply:
Yes, it will only update the acrylic in this scenario, however, it is intended to do that. This function is for background color only and whenever acrylic is enabled we don't actually draw the solid background layer, we apply the color as acrylic's tint whether or not background image is set

Reply:
So the logic is just acrylic or not. It becomes a bit confusing because of this workaround.
```cpp
// Set the default background as transparent to prevent the
// DX layer from overwriting the background image
 _settings.DefaultBackground(ARGB(0, R, G, B));
```

Reply:
Oh I'm more thinking: what happens when you change the background image url for a profile that has a background image+acrylic? Does the image change appropriately?

Reply:
Yes, it changes appropriately, although the code for it is around L251 not here.

Reply:
Hrm.  Does the alpha for the DX renderer _ever_ need to be non-zero?

Reply:
Just to clarify my previous response:

- In all cases, the background is drawn in Xaml space by the brush.
- In every case except a solid-color background (with no other modifications), the DX rendering drawing _anything_ over it is erroneous.
- Even in the solid-color background case,  the brush drawing the background (as opposed to the DX renderer) is necessary because the DX renderer doesn't draw to the entire window area.

So I'm not sure there's a good reason to have the DX renderer aware of any sort of background information, since it can't "help".

Reply:
In this specific case (Windows Terminal)*, I'm entirely inclined to agree. The default background doesn't need a color. @miniksa nominally owns the DX renderer, so he likely has a stronger opinion here.

* _dxrenderer can be used by conhost, too, and conhost has a slightly different set of rendering requirements_